src/Console/Command/ScanCommentedCodeCommand.php

```diff
@@ -36,23 +36,23 @@ protected function configure()
             ->addOption('exclude-dirs', null, InputOption::VALUE_OPTIONAL, 'Directories to exclude.')
             ->addOption('width', null, InputOption::VALUE_OPTIONAL, 'Width of the report.')
             ->addOption('show', null, InputOption::VALUE_NONE, 'If option is given description is shown.')
-            ->addOption('basedir', null, InputOption::VALUE_REQUIRED, 'The project basedir to find phpcs.', $properties['basedir']);
+            ->addOption('project.basedir', null, InputOption::VALUE_REQUIRED, 'The project basedir to find phpcs.', $properties['project.basedir']);
         // @codingStandardsIgnoreEnd
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $properties = array(
-            'basedir' => 'todo',
+            'project.basedir' => 'todo',
         );
         $dirname = !empty($input->getOption('directory')) ? $input->getOption('directory') : getcwd();
         // @codingStandardsIgnoreLine
         $exclude_dirs = !empty($input->getOption('exclude-dirs')) ? '--ignore=' . $input->getOption('exclude-dirs') . ' ' : '';
-        $basedir = !empty($input->getOption('basedir')) ? $input->getOption('basedir') : $properties['basedir'];
+        $basedir = !empty($input->getOption('project.basedir')) ? $input->getOption('project.basedir') : $properties['project.basedir'];
         //$width = !empty($input->getOption('width')) ? $input->getOption('width') : 80;
         $show = $input->getOption('show') ? true : false;
         ob_start();
-        $executable = $basedir . "/ssk/phpcs";
+        $executable = $basedir . "/toolkit/phpcs";
```

Comment:
Maybe ```${toolkit.dir.bin}/phpcs``` can simplify and remove the hardcoded `toolkit` reference

Reply:
👍